src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -77,35 +92,40 @@ class DropdownControl extends React.Component<Props> {
     );
   }
 
-  render() {
-    const {
-      children,
-      alwaysRenderMenu,
-      alignRight,
-      menuWidth,
-      blendWithActor,
-      priority,
-      className,
-    } = this.props;
+  renderChildren(isOpen: boolean, getMenuProps: GetMenuPropsFn) {
+    const {children, alignRight, menuWidth, blendWithActor, priority} = this.props;
+
+    if (typeof children === 'function') {
+      return children({isOpen, getMenuProps});
+    }
```

Comment:
We have utility for this:
https://github.com/getsentry/sentry/blob/488044c4573d9403976b5ec0978af7a3609593d9/src/sentry/static/sentry/app/utils/callIfFunction.tsx#L1-L4

Reply:
Yes I know this function, but I also remember Evan saying something about not using it, so I didn't use it

Reply:
I don't think callIfFunction is a good fit here as you still need to use typeof to know if you can return.